test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Verb.Tests.ps1

```diff
@@ -0,0 +1,48 @@
+﻿Describe "Get-Verb" -Tags "CI" {
+
+    It "Should get a list of Verbs" -test {
+        Get-Verb | Should not be $null
+    }
+
+    It "Should get a specific verb" -test {
+        (Get-Verb -Verb Add | Measure-Object).Count | Should be 1
+        (Get-Verb -Verb Add -Group Common | Measure-Object).Count | Should be 1
+    }
+
+    It "Should get a specific group" -test {
+        Get-Verb -Group Common | Should not be $null
+    }
+
+    It "Should not return duplicate Verbs with Verb paramater" -test {
+        $dups = Get-Verb -Verb Add,ad*,a*
+        $unique = $dups | 
+            Select-Object -Unique
+        $dups.Count | Should be $unique.Count 
+    }
+
+    It "Should not return duplicate Verbs with Group paramater" -test {
+        $dupVerbs = Get-Verb -Group other,other
```

Comment:
If the parameter was an enum with flags, you wouldn't need any special code to cover this case, we wouldn't see the group specified multiple times.